ext/drmgr/drmgr.h

```diff
@@ -209,13 +209,30 @@ typedef struct _drmgr_priority_t {
     int priority;
 } drmgr_priority_t;
 
+/** Specifies the callbacks when registering all \p drmgr's events */
+typedef struct _drmgr_events_t {
+    /** The size of the drmgr_events_t struct */
+    size_t struct_size;
+    /** Callback for the app2app event */
```

Comment:
Maybe link to the main docs for each of these?

Reply:
Not sure which docs you refer to. I would only think of https://dynamorio.org/page_drmgr.html#sec_drmgr_stages as a generic reference to the stages and I cannot think of any documentation for each individual stage. From other code, I would only think to point to the individual register calls (e.g., drmgr_register_bb_app2app_event)

Reply:
I'm thinking of the individual register calls.  But if you think these are well enough documented in the function that takes in this struct then could leave as is.

Reply:
Thanks for quickly clarifying. The function that uses the struct indeed points to the documentation for the individual register calls. So that should probably be sufficient.